src/components/HTMLEngineProvider/HTMLRenderers/PreRenderer/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import withLocalize from '../../../withLocalize';
+import htmlRendererPropTypes from '../htmlRendererPropTypes';
+import BasePreRenderer from './BasePreRenderer';
+
+class PreRenderer extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.wheelEvent = this.wheelEvent.bind(this);
+        this.ref = React.createRef(null);
+    }
+
+    componentDidMount() {
+        this.ref.current
+            .getScrollableNode()
+            .addEventListener('wheel', this.wheelEvent);
+    }
+
+    componentWillUnmount() {
+        this.ref.getScrollableNode()
+            .removeEventListener('wheel', this.wheelEvent);
+    }
+
+    wheelEvent(event) {
```

Comment:
Rename this function. The function name should define what it does instead of where it 
`scrollNode`.

Reply:
@parasharrajat I changed the function name to scrollNode